app/assets/stylesheets/content/editor/_ckeditor.sass

```diff
@@ -74,3 +74,37 @@
   display: block
   max-width: 400px
   @include text-shortener
+
+// Document editor styles
+.document-editor
+  /* This element is a flex container for easier rendering. */
+  display: flex
+  flex-flow: column nowrap
+
+  .document-editor__toolbar
+    position: sticky
+    top: 0
```

Comment:
Please add `z-index: 1` here. Otherwise, tables and images will be placed above the toolbar.